### ISSUE TITLE
Use .pivot() on LazyFrame

### ISSUE THREAD

**Issue by as1313** [pola-rs/polars#11045]
### Description

I am trying to use the .pivot() function on a LazyFrame dataset, but I get the following error: 
**"AttributeError: 'LazyFrame' object has no attribute 'pivot'"**

Here is a code example:
```
df = pl.DataFrame(
    {
        "foo": ["one", "one", "two", "two", "one", "two"],
        "bar": ["y", "y", "y", "x", "x", "x"],
        "baz": [1, 2, 3, 4, 5, 6],
    }
)
df = df.lazy()

df.pivot(values="baz", index="foo", columns="bar", aggregate_function="sum")
```

Could this be implemented?

**Comment by cmdlineluser** [comment]
> Pivot will not be exposed in lazy because we cannot determine the schema without collecting the data first. So you must do the collect explicit.

https://github.com/pola-rs/polars/issues/5473#issuecomment-1311320959

**Comment by as1313** [comment]
Thanks @cmdlineluser

**Comment by SamuelAllain** [comment]
[polars.LazyFrame.pivot](https://docs.pola.rs/api/python/stable/reference/lazyframe/api/polars.LazyFrame.pivot.html) is now available, starting from Python Polars 1.36.0 (2025/12). See this [pull request](https://github.com/pola-rs/polars/pull/25016)
